vitest.config.ts

```diff
@@ -39,6 +40,7 @@ export default defineConfig(
 			]
 		},
 		test: {
+			environment: 'jsdom',
 			globals: true,
 			watch: false,
 			silent: true
```

Comment:
The only doubt I would have is to add here `setupFiles: ['./vitest.setup.ts']` and create the file `vitest.setup.ts` that inside has simply:

```typescript
import '@testing-library/jest-dom';
```

The reason is because there are some functions, like `expect(element).toBeInTheDocument()` that belongs to it, so we would need to import it in each test module for the components.
Not an issue, just a redundancy I think.

Reply:
Good point!

There seems to be no other path than having a separated setup files, so added it to this PR and also added only the necessary to the TS config.